mobile_files/package.json

```diff
@@ -30,9 +29,9 @@
     "prop-types": "15.6.0",
     "punycode": "1.4.1",
     "querystring-es3": "0.2.1",
-    "react": "16.2.0",
+    "react": "16.4.1",
     "react-dom": "16.2.0",
-    "react-native": "github:status-im/react-native#status-0.55.4",
+    "react-native": "github:yenda/react-native#status-0.56",
```

Comment:
Are we switching to yenda/react-native fork?

Reply:
no I didn't had the rights to create the branch on status-im fork, but will do before rebasing. It doesn't matter for testing